em-app/src/lib.rs

```diff
@@ -263,6 +291,34 @@ fn get_attestation_certificates(client: &mut Client, report: &Report, csr_pem: S
     Ok((fqpe_cert, node_cert))
 }
 
+fn get_user_data(pub_key: &Vec<u8>, config_id: Option<&str>) -> Result<[u8;64]> {
```

Comment:
This function is taken over from `em-app 0.3` almost literally. The only difference is that errors get mapped to the `TargetReportHash` variant instead of a new `ConfigIdIssue` error variant, because adding such a variant would not be semver-compatible with the 0.2 branch.